FrejaEidClient/src/main/java/com/verisec/frejaeid/client/service/AuthenticationService.java

```diff
@@ -101,4 +119,30 @@ private boolean isPollingTimeExpired(long pollingEndTime) {
         return (System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime;
     }
 
+    public static byte[] readAllBytes(InputStream inputStream) throws IOException {
+        final int bufLen = 4 * 0x400; // 4KB
+        byte[] buf = new byte[bufLen];
+        int readLen;
+        IOException exception = null;
+
+        try {
+            try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
+                while ((readLen = inputStream.read(buf, 0, bufLen)) != -1)
+                    outputStream.write(buf, 0, readLen);
```

Comment:
I usually use brackets even when there's only one statment in a while block. It makes it more error prone when somebody else changes code, it cannot accidentally change order of line and make something else execute in the while loop.

Reply:
Done.